plugins/tmux/tmux.plugin.zsh

```diff
@@ -93,6 +101,13 @@ if which tmux &> /dev/null
 			_zsh_tmux_plugin_run
 		fi
 	fi
+
+	# Automatically refresh tmux environments if tmux is running.
+	if [[ -n "$TMUX" && "$ZSH_TMUX_AUTOREFRESH" == "true" ]]
```

Comment:
I would suggest changing this to:

```shell
if [[ -n "$TMUX"  && "$ZSH_TMUX_AUTOREFRESH" == "true" ]] && tmux ls >/dev/null 2>/dev/null; then
```

This prevents the plugin from running in an ssh shell that is a child of tmux, and also complies with the oh-my-zsh coding style guidelines.

Reply:
Done.